src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -107,26 +108,31 @@ def validate_ip_ranges(namespace):
 
     restrict_traffic_to_agentnodes = "0.0.0.0/32"
     allow_all_traffic = ""
-    ip_ranges = [ip.strip() for ip in namespace.api_server_authorized_ip_ranges.split(",")]
+    ip_ranges = [ip.strip()
+                 for ip in namespace.api_server_authorized_ip_ranges.split(",")]
 
     if restrict_traffic_to_agentnodes in ip_ranges and len(ip_ranges) > 1:
         raise CLIError(("Setting --api-server-authorized-ip-ranges to 0.0.0.0/32 is not allowed with other IP ranges."
                         "Refer to https://aka.ms/aks/whitelist for more details"))
 
     if allow_all_traffic in ip_ranges and len(ip_ranges) > 1:
-        raise CLIError("--api-server-authorized-ip-ranges cannot be disabled and simultaneously enabled")
+        raise CLIError(
+            "--api-server-authorized-ip-ranges cannot be disabled and simultaneously enabled")
```

Comment:
Please use the specific error type (such as `InvalidArgumentValueError` or `ArgumentUsageError`) instead of `CLIError`

Reply:
This is not my change, but when saving file, my linter automatically formatted it with extra line.

Reply:
I wouldn't change the error type here, since it is not related to my change. I want to make this PR clean and simple.

Reply:
OK, please submit a separate PR to refine the code here